manifests/service.pp

```diff
@@ -5,30 +5,30 @@
 # == Authors
 #
 # * Trevor Vaughan <mailto:tvaughan@onyxpoint.com>
-#
+
 class sssd::service {
 
-  file { '/etc/init.d/sssd':
+  file { $sssd::sssd_service_path:
```

Comment:
This is a bug and should only exist on EL6 systems (explains some of our recent issues actually).